.github/workflows/backend.yml

```
@@ -19,17 +19,30 @@ jobs:
   files-changed:
     name: Check which files changed
     runs-on: ubuntu-22.04
-    timeout-minutes: 3
+    timeout-minutes: 10
     outputs:
       backend_all: ${{ steps.changes.outputs.backend_all }}
+      static_viz: ${{ steps.static_viz.outputs.static_viz}}
```

is there ever a case where static viz changes and we don't want to run backend tests? If not, I think we could simplify this with
```suggestion
      backend_all: ${{ steps.changes.outputs.backend_all || steps.static_viz.outputs.static_viz }}
```

---

There were a couple of spots near the bottom where we were checking for false instead of true. Also, I think these return strings, not booleans. I see what you're getting at though